ncm-spma/src/main/pan/components/spma/schema.pan

```diff
@@ -63,7 +66,7 @@ type component_spma_type = {
     include component_spma_ips
     include component_spma_yum
     "cmdfile"       : string = "/var/tmp/spma-commands" # where to save commands for spma-run script
-    "packager"      : string = "yum" with match (SELF, '^(yum|ips)$') # system packager to be used (yum,ips)
+    "packager"      : string = "yum" with match (SELF, '^(yum|ips|rpm)$') # system packager to be used (yum,ips,rpm)
```

Comment:
We (MS) are still using rpm packager settings for legacy builds. The 'yumng' is just a subplugin of 'yum' packager. In fact we are using PACKAGER == 'yum' conditionals to check for yum base OS builds and redefining PACKAGER to subplugin would introduce unwanted high granularity. I mean having huge and complicated conditions in templates.